templates/semantic-ui/zenodo_rdm/records/detail.html

```diff
@@ -69,6 +69,14 @@ <h4 class="inline">{{ record.ui.access_status.title_l10n }}</h4>
 {%- endblock record_files -%}
 
 
+{# Additional details #}
+{%- block record_details -%}
```

Comment:
doesn't it completely override the whole block in the invenio_app_rdm/records/detail.html ? was that intentional? because I see there is quite some information displayed, and I think it will be ommitted by this change. See https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/details.html
if I am wrong, please ignore the comment

Reply:
Yes, and no. It replaces the app-rdm template with the zenodo-rdm template I added here, which is extending the app-rdm template and adds the extra content in the new `extras` block I added in the app-rdm PR https://github.com/inveniosoftware/invenio-app-rdm/pull/2520, so all the content is still there.

Reply:
I think it can stay like this for now, I am a bit confused with the hierarchy but I think it is because originally organised this way. 
Thanks for checking and testing!